src/virt-install

```diff
@@ -15,12 +15,15 @@ def fatal(msg):
     print('error: {}'.format(msg), file=sys.stderr)
     raise SystemExit(1)
 
+# Set of valid keys in image.yaml
+IMAGE_CONF_KEYS = ['size', 'inject-core-user', 'postprocess-script']
+
 parser = argparse.ArgumentParser()
 parser.add_argument("--dest", help="Destination disk",
                     action='store', required=True)
 parser.add_argument("--create-disk", help="Automatically create disk as qcow2, parsing kickstart for size",
                     action='store_true')
-parser.add_argument("--image-config", help="coreos-assembler image.yaml",
```

Comment:
i don't quite understand this change. It doesn't appear to be used anywhere in the latest iteration of the PR. Was it used earlier?

Reply:
oh actually I see.. postprocess script is not inline

Reply:
Is that something we should/could consider? i.e. the postprocess for rpm-ostree manifest is inline, which I kind of like unless it gets super long.

Reply:
I think the difference here is that doing an external script is likely going to require some nontrivial code (e.g. at least importing our not-officially-API `libguestfs.sh`) and so in contrast to things one may do in rpm-ostree like just `rm /usr/share/blah` it will end up longer.

Reply:
I see - yeah sourcing in libraries is going to be hard, but how is that achieved now with this interface? i.e. how does the postprocess script know how to source in libguestfs.sh - does it hardcode use `/usr/lib/coreos-assembler/` ?

Reply:
No one is required to use our `libguestfs.sh`; first the script doesn't have to be shell, also it can do whatever it wants (including `sudo` to use host mounts instead of libguestfs, copying it to a separate machine, whatever).

Reply:
> No one is required to use our `libguestfs.sh`

Right. That was the example you used so I was trying to understand how people were going to use it. I assume that sourcing directly from `/usr/lib/coreos-assembler` was the right answer but wanted to confirm.